model/job/mem_broker.go

```diff
@@ -196,6 +196,7 @@ func (b *memBroker) PushJob(db prefixer.Prefixer, req *JobRequest) (*Job, error)
 		err := limits.CheckRateLimit(db, ct)
 		if err == limits.ErrRateLimitReached {
 			joblog.WithFields(logrus.Fields{
+				"namespace":   "mem-broker",
```

Comment:
Well, it is `nspace` in fact: https://github.com/cozy/cozy-stack/blob/master/pkg/logger/logger.go#L88

Reply:
Woops :x

However, `joblog` already has a namespace: https://github.com/cozy/cozy-stack/blob/3d3dc465738247292ca0f9657dff888f9be13aa0/model/job/broker.go#L116

Should I change it ?

Reply:
OK, let's keep the namespace from joblog